docs/demo.md

```diff
@@ -40,7 +40,7 @@ By clicking on a row in the above table you will be brought to the corresponding
 
 ## GraphKB
 
-The demo instance of IPR is at [pori-demo.bcgsc.ca/graphkb](https://pori-demo.bcgsc.ca/graphkb). This will bring you to the login page of the GraphKB client application. The `iprdemo` credentials will also work with GraphKB to demonstrate single sign on. However, we also provide an admin demo account for GraphKB so that users can explore the admin-only sections of the application. The credentials for the administrative account are
+The demo instance of IPR is at [pori-demo.bcgsc.ca/graphkb](https://pori-demo.bcgsc.ca/graphkb). This will bring you to the login page of the GraphKB client application. The `pori_admin` credentials will also work with GraphKB to demonstrate single sign on.
```

Comment:
unless we update the demo instance this won't be true so it might be better to leave the demo page as is until we have, or are ready, to do that

Reply:
ok. I updated the branch accordingly, ty